config/app/com/microsoft/azure/iotsolutions/uiconfig/services/StatusService.java

```diff
@@ -0,0 +1,119 @@
+    private final String authName = "Auth";
+    private final String telemetryName = "Device Telemetry";
+    private final String simulationName = "Device Simulation";
+
+    private final boolean ALLOW_INSECURE_SSL_SERVER = true;
+    private final int timeoutMS = 10000;
+
+    private final WsRequestBuilder wsRequestBuilder;
+    private final IServicesConfig servicesConfig;
+
+    private static final Logger.ALogger log = Logger.of(StatusService.class);
+
+    @Inject
+    StatusService(
+        IServicesConfig servicesConfig,
+        WsRequestBuilder wsRequestBuilder) {
+        this.wsRequestBuilder = wsRequestBuilder;
+        this.servicesConfig = servicesConfig;
+    }
+
+    public StatusServiceModel getStatus() {
+        StatusServiceModel result = new StatusServiceModel(true, "Alive and well!");
+        ArrayList<String> errors = new ArrayList<String>();
+
+        // Check connection to Auth
+        StatusResultServiceModel authResult = this.PingService(
+            authName,
+            this.servicesConfig.getUserManagementApiUrl());
+        SetServiceStatus(authName, authResult, result, errors);
+        result.addProperty("UserManagementApiUrl", this.servicesConfig.getUserManagementApiUrl());
+
+        // Check connection to StorageAdapter
+        StatusResultServiceModel storageAdapterResult = this.PingService(
+            storageAdapterName,
+            this.servicesConfig.getStorageAdapterApiUrl());
+        SetServiceStatus(storageAdapterName, storageAdapterResult, result, errors);
+        result.addProperty("StorageAdapterApiUrl", this.servicesConfig.getStorageAdapterApiUrl());
+
+        // Check connection to Device Telemetry
+        StatusResultServiceModel telemetryResult = this.PingService(
+            telemetryName,
+            this.servicesConfig.getTelemetryApiUrl());
+        SetServiceStatus(telemetryName, telemetryResult, result, errors);
+        result.addProperty("DeviceTelemetryApiUrl", this.servicesConfig.getTelemetryApiUrl());
+
+        // Check connection to Device Simulation
+        StatusResultServiceModel deviceSimulationResult = this.PingService(
+            simulationName,
+            this.servicesConfig.getDeviceSimulationApiUrl());
+        SetServiceStatus(simulationName, deviceSimulationResult, result, errors);
+        result.addProperty("DeviceSimulationApiUrl", this.servicesConfig.getDeviceSimulationApiUrl());
+
+        result.addProperty("SeedTemplate", this.servicesConfig.getSeedTemplate());
+
+        if (errors.size() > 0) {
+            result.setStatus(new StatusResultServiceModel(false, String.join("; ", errors)));
+        }
+
+        log.info("Service status:" + result.getStatus().getMessage());
+
+        return result;
+    }
+
+    private void SetServiceStatus(
+        String dependencyName,
+        StatusResultServiceModel serviceResult,
+        StatusServiceModel result,
+        ArrayList<String> errors
+    ) {
```

Comment:
go through and fix issue of ) being on same line as { (it's in multiple classes)

Reply:
done